internal/server/utils/redisCmdAdapter.go

```diff
@@ -252,3 +254,58 @@ func isBase64Encoded(s string) bool {
 	}
 	return false
 }
+
+func unmarshalRequestBody(data []byte, v *map[string]interface{}) error {
```

Comment:
Since maps are already reference types, no need to send them as pointers.

Reply:
@AshwinKul28 When you pass a map to a function, Go passes a reference to the map, meaning any modifications to the contents of the map itself (adding, removing, or updating keys and values) are indeed reflected outside the function.
But, in this case, I'm trying to overwrite the entire reference itself of `v` in the last line of the method using `deoder.Decode(&v)`, basically trying to store the decoded map reference in the input map, so that the caller can directly access the map without a need for reassignment. So, passing by reference makes sense here IMO.

Reply:
@AshwinKul28 If there are no further comments, let's merge this PR?

Reply:
Hi @deydebaditya sorry got occupied with other stuff. I see one lint error which is what I was talking about earlier in this thread. Since map is a ref type no need to pass it as pointer again. 

Once you make that change hope linter checks will be successful